algrtm/src/components/PianoRollDescription.js

```diff
@@ -3,10 +3,10 @@ import React from "react";
 
 const PianoRollDescription = () => {
   return (
-    <section className="flex flex-col items-center justify-center w-full px-[12.625rem] mt-[3.94rem]">
-      <div className="flex gap-[4.5rem] items-center justify-center">
+    <section className="flex flex-col items-center justify-center w-full xl:px-[12.625rem] mt-[4rem] md:mt-[3.94rem]">
+      <div className="xl:flex-row flex flex-col gap-[4.5rem] items-center justify-center w-full">
```

Comment:
The `xl:flex-row` should be `lg:flex-row` to put the text and image on one line, once the resolution hits 1024px